.github/workflows/update-stored-dependencies-version.yaml

```diff
@@ -24,7 +24,6 @@ jobs:
           - limitador
           - limitador-operator
           - wasm-shim
-          - policy-controller
```

Comment:
I think we need to add the `dns-operator` here so the github actions variables are updated to be used in the scheduled builds

Reply:
Right, I reckon this will be also needed by QA, keeping the latest hash stored or in the end we will be moving to date based? Maybe @pehala could clarify. In the case we still need to keep track of the latest SHAs, there's also some manual work to do, which is adding the action variable in our organization account. I could show you how to or do it myself @mikenairn

Reply:
![image](https://github.com/Kuadrant/kuadrant-operator/assets/327352/3aa070af-02d8-4aed-b0c6-196c758d5611)
Added. Just used the latest commit from main.